src/addons/presenter.js

```diff
@@ -19,6 +19,54 @@ function wrappedRender () {
  */
 class Presenter extends React.Component {
 
+  static contextTypes = {
+    send : React.PropTypes.func
+  }
+
+  static childContextTypes = {
+    send : React.PropTypes.func
+  }
```

Comment:
I had to move these because IE10 doesn't forward along statics to inherited classes :(